src/Jupyter/Visualization/StateDisplayEncoders.cs

```diff
@@ -268,17 +303,65 @@ public StateVectorToHtmlResultEncoder(IConfigurationSource configurationSource)
                             _ => throw new ArgumentException($"Unsupported style {ConfigurationSource.PhaseDisplayStyle}")
                         };
 
-                        return FormattableString.Invariant($@"
-                            <tr>
-                                <td>$\left|{basisLabel}\right\rangle$</td>
-                                <td>${amplitude.Real:F4} {(amplitude.Imaginary >= 0 ? "+" : "")} {amplitude.Imaginary:F4} i$</td>
+                        count = count + 1;
```

Comment:
```suggestion
                        count += 1;
```

Reply:
In fact, as @cgranade also commented elsewhere, it may be better to just use `System.Guid.NewGuid()` instead of keeping track of a `count` variable. There is a slight danger in just using an integer like `count` to uniquely identify your HTML elements, because you don't control the entire HTML tree, and it's possible that some code somewhere else would create an element with the same ID. Using `NewGuid()` gives you a randomly-generated ID that is guaranteed to be unique, e.g., `0233e4ed-60b5-497e-bef0-ab1b0d203510`. Yes, it's long and ugly, but the JavaScript doesn't care... 😊

Reply:
Noted! Made the correction